src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt

```diff
@@ -71,4 +71,12 @@ class FeedbackDao(private val sql: DSLContext) {
             .fetchOne()?.let { Status.valueOf(it[feedbackTables.status].toString().uppercase()) }
             ?: throw BadRequestException("feedback not found")
     }
+
+    fun changeStatus(feedbackId: UUID, status: Status, activeUser: ActiveUser): Int {
+        return sql.update(feedbackTables)
+        .set(feedbackTables.status, status.toString())
+            .where(feedbackTables.feedbackId.eq(feedbackId))
+            .and(feedbackTables.companyId.eq(activeUser.companyId))
+            .execute()
+    }
```

Comment:
i'm not sure that it's the feedback of the active user that will be the one that needs the status to be updated

Reply:
Im not using tha active user id
thats the feedback id and company id